framework/cluster/pom.xml

```diff
@@ -47,8 +47,7 @@
             <groupId>org.apache.cloudstack</groupId>
             <artifactId>cloud-api</artifactId>
             <version>${project.version}</version>
-            <type>test-jar</type>
```

Comment:
Why did you remove this?

Reply:
I moved the interface `ManagementServerHost` to the API so those objects can be exposed. Therefore now there is a real dependency on the api, not only for testing.

Reply:
ok, got it.